plugins/datadog/app/models/datadog_deploy_event.rb

```diff
@@ -14,6 +14,17 @@ def self.deliver(deploy, tags:, time:)
         "error"
       end
 
+    tags += ["deploy"]
+
+    kubernetes_template =
+      defined?(SamsonKubernetes) &&
+      deploy.kubernetes? &&
+      deploy.kubernetes_release&.release_docs&.first&.resource_template&.first
```

Comment:
maybe a comment on how we assume all roles / templates will be the same / already have a validaiton to enforce that so we can do this